### PR TITLE
Build/kitalpha5

### PR DESCRIPTION
**Pull request by arnauddieumegard**

### CODE REVIEW

File: releng/plugins/org.polarsys.kitalpha.releng.targets/kitalpha.targetplatform [hunk 1]
```diff
@@ -83,9 +83,13 @@ location eclipse-release "https://download.eclipse.org/releases/2019-06/" {
...
 	org.eclipse.emf.ecoretools.design.feature.group lazy
 	org.eclipse.emf.sdk.feature.group lazy
+	org.eclipse.acceleo.doc lazy
+	org.eclipse.acceleo.source.feature.group lazy
+	org.eclipse.acceleo.feature.group lazy
+	org.eclipse.emf.cdo.feature.group lazy
 }
 
-location sirius "https://download.eclipse.org/sirius/updates/releases/6.3.0/2019-06/" {
+location sirius "https://download.eclipse.org/sirius/updates/releases/6.3.2/2019-06/" {
```
Comment (sandupostaru) [inline releng/plugins/org.polarsys.kitalpha.releng.targets/kitalpha.targetplatform:92]: It should be https://download.eclipse.org/sirius/updates/stable/6.4.0-S20200716-044517/2020-06/ in the meantime I think

Comment (arnauddieumegard) [reply releng/plugins/org.polarsys.kitalpha.releng.targets/kitalpha.targetplatform:92]: You are right

Comment (arnauddieumegard) [reply releng/plugins/org.polarsys.kitalpha.releng.targets/kitalpha.targetplatform]: Done, build in progress

**Comment by tguiu** [comment]
this PR has been split

**Pull request closed without merging**